### PR TITLE
Add method to get load of system

### PR DESCRIPTION
**Pull request by sathnaga**
Add method to get load percentage of system(host/guest)

Signed-off-by: Satheesh Rajendran <sathnaga@linux.vnet.ibm.com>

### CODE REVIEW

File: virttest/utils_hotplug.py [hunk 1]
```diff
@@ -458,3 +460,25 @@ def check_vcpu_value(vm, exp_vcpu, vcpupin=None, option="", guest_agent=False):
...
+    """
+    Get the percentage of load in Guest/Host
+    :param session: Guest Sesssion
+    :param iterations: iterations
+    :param interval: interval between load calculation
+    :return: load of system in percentage
+    """
+    idle_secs = []
+    idle_per = []
+    cmd = "cat /proc/uptime"
```
Comment (balamuruhans) [inline virttest/utils_hotplug.py]: can we reuse `utils_misc.get_uptime()` to retrieve uptime ?

Comment (sathnaga) [reply virttest/utils_hotplug.py]: @balamuruhans lets not complicate `uptime` function, this method relies on idle value, Second option from that file.

**Comment by balamuruhans** [comment]
LGTM, @sathnaga please rebase the PR.

File: virttest/utils_hotplug.py [hunk 2]
```diff
@@ -591,3 +593,25 @@ def guest_numa_check(vm, exp_vcpu):
...
+    for itr in range(iterations):
+        if session:
+            idle_secs.append(float(session.cmd_output(cmd).strip().split()[1]))
+        else:
+            idle_secs.append(float(process.system_output(cmd).split()[1]))
+        time.sleep(interval)
+    for itr in range(iterations-1):
+        idle_per.append((idle_secs[itr+1] - idle_secs[itr])/interval)
+    return int((1 - (sum(idle_per)/len(idle_per))) * 100)
```
Comment (balamuruhans) [inline virttest/utils_hotplug.py]:
```suggestion
    return int((1 - (sum(idle_per) // len(idle_per))) * 100)
```

File: virttest/utils_hotplug.py [hunk 3]
```diff
@@ -591,3 +593,25 @@ def guest_numa_check(vm, exp_vcpu):
...
+    idle_per = []
+    cmd = "cat /proc/uptime"
+    for itr in range(iterations):
+        if session:
+            idle_secs.append(float(session.cmd_output(cmd).strip().split()[1]))
+        else:
+            idle_secs.append(float(process.system_output(cmd).split()[1]))
+        time.sleep(interval)
+    for itr in range(iterations-1):
+        idle_per.append((idle_secs[itr+1] - idle_secs[itr])/interval)
```
Comment (balamuruhans) [inline virttest/utils_hotplug.py]:
```suggestion
        idle_per.append((idle_secs[itr + 1] - idle_secs[itr]) // interval)
```

**Pull request merged**